### ISSUE TITLE
torch_sparse/_convert_cpu.so: undefined symbol: _ZNK2at6Tensor6deviceEv

### ISSUE THREAD

**Issue by jingyanliao** [rusty1s/pytorch_scatter#370]
I'm trying to solve the same problem under torch 1.10.0+cu111, but after following [this solution](https://github.com/rusty1s/pytorch_scatter/issues/248#issuecomment-962952400) the problem is still unsolved.

command I used:
pip uninstall torch-scatter 
pip uninstall torch-sparse 
pip install torch-sparse -f https://data.pyg.org/whl/torch-1.10.0+cu111.html 
pip install torch-scatter -f https://data.pyg.org/whl/torch-1.10.0+cu111.html

error message:
torch_sparse/_convert_cpu.so: undefined symbol: _ZNK2at6Tensor6deviceEv

pip list
`
Package                  Version
------------------------ ------------
absl-py                  1.4.0
aiohttp                  3.8.4
aiosignal                1.3.1
argcomplete              3.0.8
async-timeout            4.0.2
attrs                    23.1.0
av                       10.0.0
av2                      0.2.1
cachetools               5.3.0
certifi                  2022.12.7
charset-normalizer       3.1.0
click                    8.1.3
cmake                    3.26.3
colorlog                 6.7.0
contourpy                1.0.7
cycler                   0.11.0
distlib                  0.3.6
filelock                 3.12.0
fonttools                4.39.3
frozenlist               1.3.3
fsspec                   2023.4.0
future                   0.18.3
google-auth              2.17.3
google-auth-oauthlib     1.0.0
googledrivedownloader    0.4
grpcio                   1.54.0
idna                     3.4
importlib-metadata       6.6.0
importlib-resources      5.12.0
isodate                  0.6.1
Jinja2                   3.1.2
joblib                   1.2.0
kiwisolver               1.4.4
lit                      16.0.2
llvmlite                 0.40.0
Markdown                 3.4.3
markdown-it-py           2.2.0
MarkupSafe               2.1.2
matplotlib               3.7.1
mdurl                    0.1.2
mpmath                   1.3.0
multidict                6.0.4
networkx                 3.1
nox                      2023.4.22
numba                    0.57.0
numpy                    1.24.3
nvidia-cublas-cu11       11.10.3.66
nvidia-cuda-cupti-cu11   11.7.101
nvidia-cuda-nvrtc-cu11   11.7.99
nvidia-cuda-runtime-cu11 11.7.99
nvidia-cudnn-cu11        8.5.0.96
nvidia-cufft-cu11        10.9.0.58
nvidia-curand-cu11       10.2.10.91
nvidia-cusolver-cu11     11.4.0.1
nvidia-cusparse-cu11     11.7.4.91
nvidia-nccl-cu11         2.14.3
nvidia-nvtx-cu11         11.7.91
oauthlib                 3.2.2
opencv-python            4.7.0.72
packaging                23.1
pandas                   2.0.1
Pillow                   9.5.0
pip                      23.0.1
platformdirs             3.5.0
protobuf                 4.22.3
pyarrow                  12.0.0
pyasn1                   0.5.0
pyasn1-modules           0.3.0
pyDeprecate              0.3.1
Pygments                 2.15.1
pyparsing                3.0.9
pyproj                   3.5.0
python-dateutil          2.8.2
python-louvain           0.16
pytorch-lightning        1.5.2
pytz                     2023.3
PyYAML                   6.0
rdflib                   6.3.2
requests                 2.30.0
requests-oauthlib        1.3.1
rich                     13.3.5
rsa                      4.9
scikit-learn             1.2.2
scipy                    1.10.1
setuptools               59.5.0
six                      1.16.0
sympy                    1.11.1
tensorboard              2.12.3
tensorboard-data-server  0.7.0
threadpoolctl            3.1.0
torch                    1.10.0+cu111
torch-geometric          1.7.2
torch-scatter            2.1.1
torch-sparse             0.6.17
torchmetrics             0.11.4
tqdm                     4.65.0
triton                   2.0.0
typing_extensions        4.5.0
tzdata                   2023.3
urllib3                  2.0.2
virtualenv               20.23.0
Werkzeug                 2.3.3
wheel                    0.38.4
yarl                     1.9.2
zipp                     3.15.0
`

**Comment by rusty1s** [comment]
Answered you in https://github.com/rusty1s/pytorch_scatter/issues/248#issuecomment-962952400.
